package.json

```diff
@@ -9,9 +9,12 @@
     "start:android:logger": "LOGGER_ENABLED=true concurrently -r 'react-native start --reset-cache' 'react-native run-android'",
     "start:ios": "concurrently -r 'react-native start' 'react-native run-ios'",
     "start:ios:logger": "LOGGER_ENABLED=true concurrently -r 'react-native start --reset-cache' 'react-native run-ios'",
+    "build:android:release": "cd android && ./gradlew assembleRelease",
     "link": "react-native link",
     "ios": "react-native run-ios",
     "android": "react-native run-android",
+    "android:release": "react-native run-android --variant=release",
+    "android:signkey": "cd android/app/ && keytool -genkey -v -keystore my-release-key.keystore -alias my-key-alias -keyalg RSA -keysize 2048 -validity 10000 && cd -",
```

Comment:
I'll explain, this can be useful if you need to create a release version of the application. Yes, you need to create a signature key, this command does this.

According to [the manual](https://facebook.github.io/react-native/docs/signed-apk-android.html), the key data must be placed in the `android/gradle.properties`.

@housseindjirdeh I really do not know how you release the new version on Google Play, this change does not hurt?